tests/test_jailed_grader.py

```diff
@@ -48,32 +48,3 @@ def zoo():
 
         response = self.g.grade(self.grader_root / 'fake_grader.py', {}, 'asdofhpsdfuh')
         self.assertEqual(response['score'], 0)
-
-    def test_correct_python3(self):
```

Comment:
why did we delete these tests?

Reply:
Because this test run the same code as in test_correct_python. Before they used codejail to run tests using python 2 and apparently python 3 separately. However since the code now supports python 2.7 and python 3.5 we just need to run the test once (using the test called test_correct_python).